src/index.test.js

```
@@ -28,11 +29,23 @@ describe('css-vendor', () => {
       expect(supportedProperty('display')).to.be('display')
     })
 
-    it('should prefix if needed', function () {
-      const {level, needPrefix} = getSupport('transforms2d')
-      if (level === 'none' || level === 'unknown') this.skip()
-      const prop = needPrefix ? `${prefix.css}transform` : 'transform'
-      expect(supportedProperty('transform')).to.be(prop)
+    it('should prefix if needed', () => {
+      const failMsgs = []
+      for (const property in propertyPrefixFixture) {
+        try {
+          expect(supportedProperty(property)).to.be(propertyPrefixFixture[property])
+        }
+        catch (e) {
+          failMsgs.push(`${e} for '${property}'`)
+        }
+      }
+      if (failMsgs.length > 0) {
+        failMsgs.forEach(err => {
+          const logMsg = `${currentBrowser.id} ${currentBrowser.version} ${err}`
+          console.log(logMsg) // eslint-disable-line no-console
```

Why do we catch the assertions and log the messages anyways? Shouldn't we just let let them throw and fix the code to support everything correctly?